Added index file to the repository

massaillon opened the pull request:
I just added the index file for the first time into the project directory

massaillon committed:
added new feature on HTML

massaillon committed:
added html file and CSS file

smunozmo commented on README.md:
```diff
@@ -1 +1,24 @@
-# Hello-Microverse
\ No newline at end of file
+Project Name
```
- [ ] Your README file is almost there, great job! Let's make it even better by adding some more information, this way your repo will look more professional, try adding these sections: 

- Contributing
- Show your support
- Acknowledgments
- License

(Here's a [template](https://github.com/microverseinc/readme-template) you can use)

massaillon replied:
Good morning sir @smunozmo 
I applied the changes based on your request
please help to review it again

Thanks

smunozmo reviewed (changes requested):
Hi @massaillon, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏 

- Great work using the right GitHub Flow.
- You have a READMEfile, well done!
- Commits are descriptive, good work.
- Your linters are working properly.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [OPTIONAL]  It is a better practice to write your commits in the imperative tense and capitalized e.g instead of `Added HTML file.` try with `Add HTML file`. You can read more about this [here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8), this will make your code look projects look more professional.

<img src="https://media1.giphy.com/media/z5cNqbFbMTDA4/giphy.gif"/>

Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

massaillon committed:
Update the README file

I updated the README file with more details based on the  first Project review   as follows:

🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the issues page.
Show your support

Give a ⭐️ if you like this project!

Acknowledgments

    Hat tip to anyone who's code was used
    Inspiration
    etc

📝 License

This project is MIT licensed.

zieeco commented on README.md:
```diff
@@ -1 +1,43 @@
+2)git and github professional best practices
+
+
+👤 Author
+
+    GitHub: @massaillon
+    LinkedIn: Massaillon Silue
+    
+🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+Show your support
+
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+
+    Hat tip to anyone whose code was used
+    Inspiration
```
- [ ]  OPTIONAL
 Great job on implementing the changes required by the previous reviewer. Below are some things to bear in mind and consider as **MANDATORY REQUIREMENT** in your next project.
 - Your PR title should be very descriptive, instead of `Added index to the repository`, you could use `Hello Microverse, greetings`.
 - Your readme authors should link to your `GitHub, LinkedIn, and Twitter account`. Do try maintaining consistency in following this [microverse readme template](https://github.com/microverseinc/readme-template)
 - Your `text.txt` should not be showing in your repo.
 - Your PR should be more descriptive pointing out what you did and summarizing all you did.

massaillon replied:
HI sir
Noted OK 
I will reconsider your suggestions
thanks for the strong support

zieeco reviewed (approved):
# APPROVED :green_circle: :100: : 

Hi @massaillon 

<img src="https://media2.giphy.com/media/3kuSo744UIPJjcJUEn/giphy.gif"/>

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
_Check the comments under the review._


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
